src/promptflow/promptflow/_sdk/_service/apis/connection.py

```diff
@@ -48,6 +53,15 @@
 )
 
 
+def get_connection_operation(working_directory=None):
```

Comment:
will create ml_client every time cause performance issue?

Reply:
No time cost and memory issues have been identified so far